packages/documentation-site/patternfly-docs/content/developer-resources/rtl-handbook.md

```diff
@@ -11,25 +11,34 @@ The default browser behavior displays content following LTR formatting. In order
+### Components 
 
-PatternFly components have been written to reverse directional icons that are part of the component - for example, the caret indicating expansion of various components is properly reversed. 
+PatternFly components have been written to use [logical properties](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_logical_properties_and_values) - `block-start`, `inline-end`, `block-end`, and `inline-start` - rather than `top`, `right`, `bottom`, and `left`. This allows styles like margin and padding to be correctly laid out in both LTR and RTL. However, to avoid a sweeping breaking change, PatternFly variable names still use `top`, `bottom`, `left`, and `right`.
 
-Other icons you may have in your interface will not be mirrored automatically, as it may be appropriate to mirror some but not others. Be sure to review all icons in your product and adjust appropriately. PatternFly has modifier classes listed below that can help with that.
-<br />
+**Note:** An exception to this support is with any components using popper.js, like tooltips and popovers. 
+
+### Charts 
+
+To support RTL across PatternFly charts, we've added a new `legendDirection` property to the
+`<Chart>`, `<ChartPie>`, `<ChartBullet>`, `<ChartDonut>`, and `<ChartDonutUtilization>` components. This property allows you to adjust the direction of chart legend labels. When in RTL, the value of this property should be set to "rtl". 
+
+Additionally, when in RTL, ensure that the `<ChartAxis>` representing a chart's x-axis uses the `invertAxis` property to flip the axis.
+
+**Note:** When using self-defined custom legends and label components, RTL support must be handled manually as `legendDirection` does not apply to custom implementations.
+
+### Icons
+PatternFly components have also been written to reverse directional icons that are part of the component. For example, the caret indicating expansion of various components is properly reversed. 
+
+Care should be taken to handle icons appropriately for RTL. Some icons you may have in your interface will not be mirrored automatically. [Sometimes an icon should be reversed in RTL](https://m2.material.io/design/usability/bidirectionality.html#mirroring-elements) (for example, when indicating a direction such as an expansion caret), and sometimes it should not be reversed (such as a logo, or an icon indicating time). When an icon *should* be reversed, but isn't reversed automatically, you can use the modifier `.pf-v5-m-mirror-inline-rtl`.
 
 ## PatternFly modifiers supporting RTL
 
-Use the modifier class `.pf-v5-m-mirror-inline-rtl` to reverse an icon or other element only when in RTL.
+-To reverse an icon or other element when in RTL, use the modifier class `.pf-v5-m-mirror-inline-rtl` or the `shouldMirrorRTL` property of the `<Icon>` React component.
 
 To set direction manually on an element, use `.pf-v5-m-dir-ltr` or `.pf-v5-m-dir-rtl`.
```

Comment:
I wonder if it's worth noting that the preferred way of defining direction on an element is via the native `dir` html attribute, eg, `dir="rtl"` or `dir="ltr"`? wdyt @srambach

Reply:
If someone uses dir for this, does that negate the need for the modifier class or the shouldMirror prop? If so - in what cases would a developer use one vs the other out of all these options?

Reply:
Great question! You would use one or the other, so no need to use `dir` and either of those classes together.

Probably the best way to suggest when to use `dir` or a class would be - the classes are meant for:

* Changing the direction of something as a stylistic/presentational-only change. 
* On the off chance a dev is unable to change the markup (to add a `dir` attribute), they can use the classes as an alternative.

Otherwise, you'd always want to use `dir` for anything where the writing direction should change for language support.

Reply:
okay, gotcha! Can you look at my recent changes and lmk if you think they effectively convey this info?